### DISCUSSION TITLE
Mixed case table columun headings

### DISCUSSION THREAD

**Discussion by lozdown** [sqlpage/SQLPage#169] [Q&A] [answered]
Is it possible to make table column headings mixed case?

**Comment by lovasoa** [comment, marked as answer]
Hello and welcome to SQLPage !
  You can rename table headings however you want by using `as`:

```sql
select 'table' as component;
select my_column_1 as "My Column 1", my_other_column as "My Other Column";
```

If you have a lot of columns, you can use the `css` top-level property of the [shell component](https://sql.ophir.dev/documentation.sql?component=shell#component) to apply a custom style to your table:

`style.css` :

```css
th {
  text-transform: capitalize !important;
}
```

`page.sql`

```sql
select 'shell' as component, 'style.css' as css;

select 'table' as component;
select * from my_table;
```

**Comment by lozdown** [comment]
Thanks - I find that both changes are required; providing the required column headings and overriding the default css
